tests/integrations/mcs/tso/server_test.go

```diff
@@ -345,3 +348,36 @@ func TestAdvertiseAddr(t *testing.T) {
 	tsoServerConf := s.GetConfig()
 	re.Equal(u, tsoServerConf.AdvertiseListenAddr)
 }
+
+func TestMetrics(t *testing.T) {
+	re := require.New(t)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	cluster, err := tests.NewTestAPICluster(ctx, 1)
```

Comment:
Can we put cluster and service initialization into suite setup and share it with other similar test cases to save test time?

Reply:
Yes, maybe do it in another PR.